From df78a9c6f099783420a62b507b805808b97c8919 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 3 Jul 2020 15:33:51 +0800
Subject: [PATCH] policy/modules/services/radius: allow radius_t to bind
 unreserved port

Fixes:
avc:  denied  { name_bind } for  pid=472 comm="radiusd" src=18120
scontext=system_u:system_r:radiusd_t:s0-s15:c0.c1023
tcontext=system_u:object_r:unreserved_port_t:s0 tclass=udp_socket
permissive=0

Upstream-Status: Pending

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/services/radius.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/services/radius.te b/policy/modules/services/radius.te
index 76dd67ed4..1accffff6 100644
--- a/policy/modules/services/radius.te
+++ b/policy/modules/services/radius.te
@@ -82,6 +82,7 @@ corenet_tcp_connect_snmp_port(radiusd_t)
 
 corenet_sendrecv_generic_server_packets(radiusd_t)
 corenet_udp_bind_generic_port(radiusd_t)
+corenet_udp_bind_all_unreserved_ports(radiusd_t)
 corenet_dontaudit_udp_bind_all_ports(radiusd_t)
 
 corecmd_exec_bin(radiusd_t)
-- 
2.17.1

